concent_api/core/tests/utils.py

```diff
@@ -0,0 +1,65 @@
+from base64                 import b64encode
+
+from django.conf            import settings
+from django.test            import TestCase
+
+from golem_messages         import dump
+from golem_messages         import message
+import dateutil.parser
+
+from utils.testing_helpers  import generate_ecc_key_pair
+
+
+class ConcentIntegrationTestCase(TestCase):
```

Comment:
This should be in a separate commit. Currently it's hidden in a commit that adds a bunch of tests.